modules/build/src/main/scala/scala/build/bsp/BspImpl.scala

```diff
@@ -1,5 +1,6 @@
 package scala.build.bsp
 
+import ch.epfl.scala.bsp4j._
```

Comment:
don't import `bsp4j._` here.
instead, use the types from there with the `b` prefix` imported on the line below.
For example, `b.TaskId`.